flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -359,3 +359,30 @@ def test_delete_an_asset(client, setup_api_test_data, requesting_user):
     assert delete_asset_response.status_code == 204
     deleted_asset = GenericAsset.query.filter_by(id=existing_asset_id).one_or_none()
     assert deleted_asset is None
+
+
+@pytest.mark.parametrize(
+    "requesting_user",
+    ["test_prosumer_user@seita.nl"],
+    indirect=True,
+)
+def test_get_assets_new(
```

Comment:
What does the "new" signify here? Would not `test_consultant_can_read` be better?

Reply:
Thanks for catching this. changed naming to `test_consultant_can_read`.